libs/asset-editor/src/lib/components/asset-editor-tab-general/asset-editor-tab-general.component.ts

```diff
@@ -41,7 +41,7 @@ const initialAssetEditorTabGeneralState: AssetEditorTabGeneralState = {
 })
 export class AssetEditorTabGeneralComponent implements OnInit {
   private readonly rootFormGroupDirective = inject(FormGroupDirective);
-  private readonly rootFormGroup: AssetEditorFormGroup = this.rootFormGroupDirective.control;
+  protected readonly rootFormGroup: AssetEditorFormGroup = this.rootFormGroupDirective.control;
```

Comment:
Why does this need to be `protected` instead of `private`?

Reply:
I references it in the tempalte temporarily, so it was just a relict